controllers/usersignup/usersignup_controller_test.go

```diff
@@ -518,9 +570,7 @@ func TestNonDefaultNSTemplateTier(t *testing.T) {
 		HasLabelWithValue(toolchainv1alpha1.MasterUserRecordOwnerLabelKey, userSignup.Name).
 		HasOriginalSub(userSignup.Spec.OriginalSub).
 		HasUserAccounts(1).
-		HasUserAccountTierName("custom").
-		HasUserAccountNamespaceTemplateRefs("custom-dev-123abc1", "custom-stage-123abc2").
-		HasUserAccountClusterResourceTemplateRefs("custom-clusterresources-654321b")
+		HasTier(*customTier)
 
```

Comment:
The same also here - we should verify that the Space was created with the `custom` tier as well

Reply:
Done in 19961f1e1623520e0ec8e74e45851b24ee4bf74a